src/components/textField/index.js

```diff
@@ -197,7 +197,8 @@ export default class TextField extends BaseInput {
       value: props.value, // for floatingPlaceholder functionality
```

Comment:
You created the branch again from another branch. 
This change/file should be in this PR

Reply:
Done.